app/src/main/kotlin/com/wire/android/ui/home/conversations/details/dialog/DeleteConversationGroupDialog.kt

```diff
@@ -17,6 +18,12 @@ internal fun DeleteConversationGroupDialog(
     isLoading: Boolean,
     onDeleteGroup: (GroupDialogState) -> Unit,
 ) {
+    LaunchedEffect(isLoading) {
+        if (!isLoading) {
+            dialogState.dismiss()
+        }
+    }
+
```

Comment:
@Garzas see this, it works and we do not have to worry about doing it manually inside the consumer